docs/source/application_development/porter.rst

```diff
@@ -0,0 +1,633 @@
+|                                  |               | | in result.                                  |
++----------------------------------+---------------+-----------------------------------------------+
+| ``exclude_ursulas`` *(Optional)* | List[Strings] | | List of Ursula checksum addresses to not    |
+|                                  |               | | include in the result.                      |
++----------------------------------+---------------+-----------------------------------------------+
+
+
+Returns
++++++++
+List of Ursulas with associated information:
+
+    * ``encrypting_key`` - Ursula's encrypting key encoded as hex
+    * ``checksum_address`` - Ursula's checksum address
+    * ``uri`` - Ursula's URI
+
+Example Request
++++++++++++++++
+.. code:: bash
+
+    curl -X GET <PORTER URI>/get_ursulas \
+        -H "Content-Type: application/json" \
+        -d '{"quantity": 5,
+             "duration_periods": 4,
+             "include_ursulas": ["0xB04FcDF9327f65AB0107Ea95b78BB200C07FA752"],
+             "exclude_ursulas": ["0x5cF1703A1c99A4b42Eb056535840e93118177232", "0x9919C9f5CbBAA42CB3bEA153E14E16F85fEA5b5D"]}'
+
+OR
+
+.. code:: bash
+
+    curl -X GET "<PORTER URI>/get_ursulas?quantity=5&duration_periods=4&include_ursulas=0xB04FcDF9327f65AB0107Ea95b78BB200C07FA752&exclude_ursulas=0x5cF1703A1c99A4b42Eb056535840e93118177232,0x9919C9f5CbBAA42CB3bEA153E14E16F85fEA5b5D"
+
+
+Example Response
+++++++++++++++++
+.. code::
+
+    Status: 200 OK
+
+
+.. code:: json
+
+    {
+       "result": {
+          "ursulas": [
+             {
+                "encrypting_key": "025a335eca37edce8191d43c156e7bc6b451b21e5258759966bbfe0e6ce44543cb",
+                "checksum_address": "0x5cF1703A1c99A4b42Eb056535840e93118177232",
+                "uri": "https://3.236.144.36:9151"
+             },
+             {
+                "encrypting_key": "02b0a0099ee180b531b4937bd7446972296447b2479ca6259cb6357ed98b90da3a",
+                "checksum_address": "0x7fff551249D223f723557a96a0e1a469C79cC934",
+                "uri": "https://54.218.83.166:9151"
+             },
+             {
+                "encrypting_key": "02761c765e2f101df39a5f680f3943d0d993ef9576de8a3e0e5fbc040d6f8c15a5",
+                "checksum_address": "0x9C7C824239D3159327024459Ad69bB215859Bd25",
+                "uri": "https://92.53.84.156:9151"
+             },
+             {
+                "encrypting_key": "0258b7c79fe73f3499de91dd5a5341387184035d0555b10e6ac762d211a39684c0",
+                "checksum_address": "0x9919C9f5CbBAA42CB3bEA153E14E16F85fEA5b5D",
+                "uri": "https://3.36.66.164:9151"
+             },
+             {
+                "encrypting_key": "02e43a623c24db4f62565f82b6081044c1968277edfdca494a81c8fd0826e0adf6",
+                "checksum_address": "0xfBeb3368735B3F0A65d1F1E02bf1d188bb5F5BE6",
+                "uri": "https://128.199.124.254:9151"
+             }
+          ]
+       },
+       "version": "6.0.0"
```

Comment:
What is this the version of? `nucypher`?

Reply:
Yes, it is the version of nucypher. I'm just proactively setting the version to 6.0.0 in the docs so that I don't have to update it later.